### PR TITLE
Add search by affiliation and topic

### PR DESCRIPTION
**Pull request by breuleux**
* Add search by affiliation to the interface
* Add search by topic to the interface
* The paper HTML is now sent by batch to the interface, so it's faster without being overwhelming
* Properly show the total count of found papers and not just the number shown

### CODE REVIEW

File: paperoni/webapp/common.py [hunk 1]
```diff
@@ -437,20 +459,33 @@ def _soft_restart(new_gen):
...
                 self.page[self.wait_area].set("✓")
-                continue
 
-            yield element
+            if not done_showing:
+                for element in elements:
+                    yield element
+
+            if done:
+                done_showing = True
```
Comment (satyaog) [inline paperoni/webapp/common.py:488]: Is it possible for

```python
            if count > self.elements["limit"].value:
                done_showing = True
```

to stop the print of the elements if for exemple we have printed 45 elements on a limit of 50 with a new batch of 15? `count > self.elements["limit"].value` would be `True` and the last 5 elements to print would be skipped?

```python
            if not done_showing:
                for element in elements:
                    yield element
```

**Review by satyaog — COMMENTED** [review]
Is it expected to have the content of #49 in this PR?

**Comment by breuleux** [comment]
> Is it expected to have the content of #49 in this PR?

Yeah it was based on #49 but now it's merged so only the difference should appear.

File: paperoni/webapp/common.py (hunk 1 above)
Comment (breuleux) [reply paperoni/webapp/common.py:488]: You're right, it would. It's minor, but I'll fix it.

Comment (breuleux) [reply paperoni/webapp/common.py]: Fixed in commit 4de010d

**Pull request merged**